### DISCUSSION TITLE
Docker image will be removed.

### DISCUSSION THREAD

**Discussion by janiversen** [pymodbus-dev/pymodbus#1612] [Help] [answered]
Unless somebody volunteers to maintain the build process.

Docker was added, because a contributor wanted it, had the time to add it, and wanted to maintain it.

However it is slowly bit rotting, a couple of months ago, signing was deactivated, and lately it have had a number of build failures.

So please if you use the docker image, please volunteer to maintain it.

**Comment by janiversen** [comment, marked as answer]
Removed on dev.

**Reply by dhoomakethu** [reply]
Yes, this was the reason I was against having it or for that matter anything that could be a maintenance overhead. Its easier to build a docker image and any one in need can build their own.

**Reply by janiversen** [reply]
you are completely right, I just hoped we had gained another contributor, but it turned out to be a wrong hope.
